### DISCUSSION TITLE
tickets for quiz teams

### DISCUSSION THREAD

**Discussion by bmaster001** [pretix/pretix#4435] [Q&A]
We would like to use pretix (self hosted) to sell tables for a quiz we are organizing. 
I added a custom question so that a team can enter the team-name during the sale. 
Besides that, each team needs a table-number (the physical table where they will be seated). No idea how to do this.... I was hoping to use a second question for this, but one that's auto filled in with an auto incremented number but I don't think it's possible.
Anyone here with a suggestion for this?

**Comment by raphaelm** [comment]
The only thing we really have for that is this Enterprise plugin: https://marketplace.pretix.eu/products/seating/
